MAUI/DataGrid/Column-types.md

```diff
@@ -947,6 +997,20 @@ dataGrid.Columns.Add(numericColumn);
 
 {% endtabs %}
 
+### Number formatting
+
+The `DataGridNumericColumn` allows formatting the numeric data with culture-specific information.
+
+* `Minimum` - To set the minimum value for the numeric column, use the `DataGridNumericColumn.Minimum` property.
+
+* `Maximum` - To set the maximum value for the numeric column, use the `DataGridNumericColumn.Maximum` property.
+
+* `ShowClearButton` - To clear the value for the numeric column, use the `DataGridNumericColumn.ShowClearButton` property.
+
+* `Placeholder` - To set the placeholder when the numeric cell value was null, use the `DataGridNumericColumn.Placeholder` property.
+
+* `NullValue` - To set the null value when the numeric cell value was null, use the `DataGridNumericColumn.NullValue` property.
```

Comment:
To set the null value when the numeric cell value is null, use the `DataGridNumericColumn.NullValue` property.

Reply:
Changed